sources/packages/api/src/database/entities/student-user.model.ts

```diff
@@ -0,0 +1,34 @@
+import { Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from "typeorm";
+import { RecordDataModel } from ".";
+import { ColumnNames, TableNames } from "../constant";
+import { Student } from "./student.model";
+import { User } from "./user.model";
+
+/**
+ * Students and users relationships, current and past ones.
+ * Every time that a a student/user association changes this
```

Comment:
two `a a`

Reply:
Please remove "that".

Reply:
I can remove "that" but, is it wrong?

Reply:
Not wrong but also the grammatical syntax is not right.

Reply:
Fair enough, I will change it.